lib/dry/web/roda/templates/view__context.rb.tt

```diff
@@ -26,16 +22,6 @@ module <%= config[:camel_cased_app_name] %>
       def flash?
         %i[notice alert].any? { |type| flash[type] }
       end
-
-      def with(new_attrs)
-        self.class.new(attrs.merge(new_attrs))
-      end
-
-      private
-
-      def [](name)
-        attrs.fetch(name)
```

Comment:
you need those method or some other way to fetch `_@options`
so it should be like

```ruby
def [](name)
  @options.fetch(name)
end
```